src/condition.cpp

```diff
@@ -2002,7 +2000,7 @@ bool ConditionManaShield::startCondition(Creature* creature)
 		return false;
 	}
 
-	if (Player* player = creature->getPlayer()) {
+	if (auto player = creature->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
	if (const auto player = creature->getPlayer()) {
```